--- a/RMVtransport/rmvtransport.py	2026-06-07 09:56:25.983818063 +0200
+++ b/RMVtransport/rmvtransport.py	2026-06-07 09:57:36.763882051 +0200
@@ -1,4 +1,5 @@
 """A module to query bus and train departure times."""
+
 import asyncio
 import json
 import logging
@@ -7,7 +8,6 @@
 from datetime import datetime
 from typing import Any, Dict, List, Optional, Union
 
-import async_timeout
 import httpx
 from lxml import etree, objectify  # type: ignore
 
@@ -148,7 +148,7 @@
 
     async def _query_rmv_api(self, url: str) -> Any:
         """Query RMV API."""
-        async with async_timeout.timeout(self._timeout):
+        async with asyncio.timeout(self._timeout):
             async with httpx.AsyncClient() as client:
                 try:
                     response = await client.get(url)
