--- a/pyproject.toml	2026-05-20 20:18:14.776284834 +0200
+++ b/pyproject.toml	2026-05-20 20:17:35.231360578 +0200
@@ -36,6 +36,7 @@
 [build-system]
 # pycares required because of the fact that it's in the __init__ module
 requires = ["setuptools", "pycares"]
+build-backend = "setuptools.build_meta"
 
 [tool.setuptools]
 # Disables inclusion of non-package data (like tests) in the wheel if they are in the sdist
