--- a/setup.py	2024-04-07 15:39:36.000000000 +0200
+++ b/setup.py	2025-09-13 16:46:35.527852465 +0200
@@ -4,20 +4,6 @@
 
 from setuptools import find_packages, setup
 
-# Get setup packages from requirements.txt
-with open("requirements_setup.txt", encoding="utf-8") as f:
-    requirements_setup = f.read().splitlines()
-
-# Get packages from requirements.txt
-with open("requirements.txt", encoding="utf-8") as f:
-    requirements = f.read().splitlines()
-
-copytree(
-    "out",
-    "systembridgefrontend/out",
-    dirs_exist_ok=True,
-)
-
 package_data = ["out/**/*"]
 
 for root, _, files in os.walk("systembridgefrontend/out"):
@@ -43,10 +29,8 @@
     long_description=readme,
     long_description_content_type="text/markdown",
     url="https://github.com/timmo001/system-bridge-frontend",
-    install_requires=requirements,
     packages=find_packages(exclude=["tests", "generator"]),
     package_data={"": package_data},
     python_requires=">=3.11",
-    setup_requires=requirements_setup,
     use_incremental=True,
 )
