--- ./src/data/cover.ts.old	2026-08-03 07:09:35.940395997 +0200
+++ ./src/data/cover.ts	2026-08-03 07:10:24.987200211 +0200
@@ -10,7 +10,7 @@
 
 export { CoverEntityFeature };
 
-export const DEFAULT_COVER_FAVORITE_POSITIONS = [0, 25, 75, 100];
+export const DEFAULT_COVER_FAVORITE_POSITIONS = [0, 25, 50, 75, 100];
 
 export const coverSupportsPosition = (stateObj: CoverEntity) =>
   supportsFeature(stateObj, CoverEntityFeature.SET_POSITION);
